docs/deployment/managed-scanning.md

```diff
@@ -11,53 +11,53 @@ tags:
+- To receive Supply Chain findings, you must have a supported lockfile in your repository. Managed Scans does **not** support lockfile generation.
+- For existing Semgrep projects, custom `semgrep.yml` configurations are not copied or detected when you use Managed Scans. If you have additional build steps when scanning, use [Semgrep in CI instead](/deployment/add-semgrep-to-ci).
 :::
 
 ## Security
 
-Managed scanning requires **[<i class="fas fa-external-link fa-xs"></i> read access](https://docs.github.com/en/rest/authentication/permissions-required-for-github-apps?apiVersion=2022-11-28#repository-permissions-for-projects)** to your code in GitHub for the repositories you choose to scan. Semgrep clones your repository at the beginning of every scan. Once the scan completes, the clone is destroyed and is not persisted anywhere.
+Managed scans require **[<i class="fas fa-external-link fa-xs"></i> read access](https://docs.github.com/en/rest/authentication/permissions-required-for-github-apps?apiVersion=2022-11-28#repository-permissions-for-projects)** to your code in GitHub for the repositories you choose to scan. Semgrep clones your repository at the beginning of every scan. Once the scan completes, the clone is destroyed and is not persisted anywhere.
 
 The access to your code is facilitated by a **private Semgrep GitHub app** that you create and register in your GitHub organization.
 
 - You are in control of the app and can [revoke access to repositories](#remove-the-private-app) at any time.
 - You can [limit access to specific repositories](#limit-access-to-specific-repositories).
 
-Managed scanning is specifically designed to limit the amount of time that code remains on our infrastructure.
+Managed scans are specifically designed to limit the amount of time that code remains within Semgrep infrastructure.
 
 ### Life cycle of a managed scan
 
 1. When a scan begins, Semgrep creates an ephemeral container and clones the repository into it.
 1. Semgrep runs the scan from that container. Diff-aware scans typically take seconds, while full scans can take minutes to hours to complete.
 1. The ephemeral container is immediately and automatically destroyed post-scan along with all contents in it.
 
-## Add a repository to Semgrep Managed Scanning
+## Add a repository to Semgrep Managed Scans
 
 Part of the process of adding a repository includes creating and registering a public Semgrep GitHub app and a private Semgrep GitHub App.
 
 - The public Semgrep GitHub app is required to easily add members of your GitHub org to your Semgrep org.
-- The private Semgrep GitHub app is required to enable code access for managed scanning.
+- The private Semgrep GitHub app is required to enable code access for managed scans.
```

Comment:
```suggestion
- The private Semgrep GitHub app is required to enable code access for Managed Scans.
```
hmm is there some difference between Managed Scans the product and... managed scans the scan? 🤔

Reply:
Yes and in this case your suggestion is correct! :D